File: Frontend/student_initializer/lib/data_old/remote/models/learner.dart
class LearnerModel {
  int? learnerId;
  String? firstName;
  String? lastName;

  LearnerModel({this.learnerId, this.firstName, this.lastName});

  LearnerModel.fromJson(Map<String, dynamic> json) {
    learnerId = json['learnerId'];
    firstName = json['firstName'];
    lastName = json['lastName'];
  }
}

File: Backend/llm-interface/src/main/java/org/ba/infrastructure/restclient/dto/Learner.java
package org.ba.infrastructure.restclient.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Learner {
    private Long learnerId;
    private String firstName;
    private String lastName;
}